examples/webgpu_video_frame.html

```diff
@@ -43,14 +43,21 @@
 
 				const geometry = new THREE.PlaneGeometry();
 
-				const videoTexture = new THREE.VideoFrameTexture();
-				videoTexture.colorSpace = THREE.SRGBColorSpace;
+				const videoFrameTexture = new THREE.VideoFrameTexture();
+				videoFrameTexture.colorSpace = THREE.SRGBColorSpace;
+
+				let lastDecodedVideoFrame = null;
 
 				// eslint-disable-next-line compat/compat
 				const decoder = new VideoDecoder( {
 					output( frame ) {
 
-						videoTexture.setFrame( frame );
+						videoFrameTexture.setFrame( frame );
+
+						// close frames no longer needed to avoid decoder stalls. https://w3c.github.io/webcodecs/#dom-videodecoder-decode
+
+						if ( lastDecodedVideoFrame ) lastDecodedVideoFrame.close();
```

Comment:
Um, why did you add this comment? If closing is no longer required, why are we doing it?

Reply:
sorry for that confusion.

it should be "to avoid video decoder stalls, we should close the VideoFrame which is no longer needed"

Reply:
updated comment